packages/sanity/src/structure/getIntentState.ts

```diff
@@ -64,3 +57,11 @@ export function getIntentState(
 
   return {intent: intent, params, payload}
 }
+
+function getPaneParams(intent: string, {template, version}: Record<string, string>): {template?: string, version?: string} {
```

Comment:
This is good – think its just missing a newline at the end of the file (I'm sorry you have to battle our linter so much 😅)

Reply:
I wish it would run in vscode in github 😅